src/commands/serve/serve.ts

```diff
@@ -148,7 +148,7 @@ export const serve = async (options: OptionValues, command: BaseCommand) => {
 
   // TODO: We should consolidate this with the existing config watcher.
   const getUpdatedConfig = async () => {
-    const { config: newConfig } = await command.getConfig({ cwd: command.workingDir, offline: true, state })
```

Comment:
Not sure if it was me that committed this, but should we keep state here? And in the dev command too?

Reply:
That change was me! I've removed the `state` argument, because its only purpose was to be used in determining the site ID. I've moved that logic into an instance variable now, so it's not needed anymore.